azimuth/routers/v1/utterances.py

```diff
@@ -105,6 +105,7 @@ def get_utterances(
     dataset_split_manager: DatasetSplitManager = Depends(get_dataset_split_manager),
     pipeline_index: Optional[int] = Depends(query_pipeline_index),
     pagination: Optional[PaginationParams] = Depends(get_pagination),
+    without_postprocessing: bool = False,
```

Comment:
Sorry, I am late to the party, I didn't realize that was getting merged.
Everywhere this query parameter is defined, it should have an alias to make it camelCase, like the others, for example `smartTags`, `dataActions`, or `confidenceMin`/`Max`).
@nandhinibsn, could you add those in your PR?
```suggestion
    without_postprocessing: bool = Query(
        False, title="Get predictions without postprocessing", alias="withoutPostprocessing"
    ),
```

Reply:
I agree. I can add those alias on the UI.

Reply:
Oh so sorry, thanks! I should have tagged you in that PR.

Reply:
No-no-no, no need to be sorry. One step at a time. 👍